runner/src/runner.rs

```diff
@@ -30,6 +29,9 @@ use std::{
 
 use async_trait::async_trait;
 
+/// Name of the downloaded client executable.
+pub const EXECUTABLE_NAME: &str = "vault";
```

Comment:
The user would have to specify a different directory for a different network. But really, per-network functionality hasn't been implemented at all so far. Reusing a directory with differently-named binaries would not be great since operators might still confuse the keyfiles.